mmv1/templates/terraform/examples/scan_config_basic.tf.erb

```diff
@@ -9,8 +9,3 @@ resource "google_security_scanner_scan_config" "<%= ctx[:primary_resource_id] %>
   starting_urls    = ["http://${google_compute_address.scanner_static_ip.address}"]
   target_platforms = ["COMPUTE"]
 }
-
-provider "google-beta" {
-  region = "us-central1"
-  zone   = "us-central1-a"
-}
```

Comment:
Let's also keep this in the test?

Reply:
I tested locally without the provider configuration with `terraform apply` and it worked. It looks like the provider configuration is optional for the users.